core/lib/tracehouse/trace-processor.js

```diff
@@ -645,18 +704,37 @@ class TraceProcessor {
           parent: evt.args.data.parent,
         });
       });
+
     const frames = [...framesById.values()];
     const frameIdToRootFrameId = this.resolveRootFrames(frames);
 
+    const inspectedTreeFrameIds = [...frameIdToRootFrameId.entries()]
+      .filter(([childFrameId, rootFrameId]) => rootFrameId === mainFrameInfo.frameId) // eslint-disable-line no-unused-vars
+      .map(([child, root]) => child); // eslint-disable-line no-unused-vars
```

Comment:
nit unless you're trying to be explicit
```suggestion
      .filter(([, rootFrameId]) => rootFrameId === mainFrameInfo.frameId)
      .map(([child]) => child);
```